File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/Solution.java
// Return the difference between the product of its digits and the sum of its digits

class Solution {
    public int subtractProductAndSum(int n) {
        int pro = 1;
        int sum = 0;

        while (n > 0) {
            int dig = n % 10;
            pro *= dig;
            sum += dig;
            n /= 10;
        }

        return pro - sum;
    }
}

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/Solution.js
// Return the difference between the product of its digits and the sum of its digits

/**
 * @param {number} n
 * @return {number}
 */

var subtractProductAndSum = function (n) {
    let pro = 1;
    let sum = 0;

    while (n > 0) {
        let dig = parseInt(n % 10);
        pro = pro * dig;
        sum = sum + dig;
        n = parseInt(n / 10);
    }

    return pro - sum;
};
